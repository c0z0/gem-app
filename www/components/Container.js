import styled from 'styled-components'

export default styled.div`
  max-width: 900px;
  margin: 0 auto;
  padding: 0 12px;
`
