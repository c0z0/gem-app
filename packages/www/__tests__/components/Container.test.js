import React from 'react'
import { render } from 'react-testing-library'
import 'jest-styled-components'

import Container from '../../components/Container'

test('Container matches snapshot', () => {
  const { container } = render(<Container />)

  expect(container).toMatchSnapshot()
})
