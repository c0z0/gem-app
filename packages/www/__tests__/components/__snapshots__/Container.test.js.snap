// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container matches snapshot 1`] = `
.c0 {
  max-width: 900px;
  margin: 0 auto;
  padding: 0 12px;
}

<div>
  <div
    class="c0"
  />
</div>
`;
